kratos/mpi/python_scripts/mpi_module_init.py

```diff
@@ -14,10 +14,18 @@
     sys.setdlopenflags(dll_load_flags)
 
 from KratosMPI import *
-from .. import legacy_mpi_python_interface
 
 if sys.platform.startswith('linux'):
     # restore default system flags
     sys.setdlopenflags(flags)
 
-mpi = legacy_mpi_python_interface.LegacyMPIPythonInterface()
+def __PathInitDetail():
+    from KratosMultiphysics import KratosPaths
+    import os
+
+    kratos_mpi_python_path = os.path.abspath(os.path.join(KratosPaths.kratos_python_source_path, "kratos/mpi/python_scripts"))
```

Comment:
I guess the general alternative would be 
```suggestion
    kratos_mpi_python_path = os.path.abspath(os.path.join(KratosPaths.kratos_python_source_path, "kratos", "mpi", "python_scripts"))
```
but not sure if it is necessary bcs this most probably will never run on Win :)

Reply:
It is not necessary, windows (python in windows?) understands the `/` for folders (the kratos loader used them for years and nobody complained). Accepting anyway because I like the OS-agnostic notation.